opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -1463,6 +1476,15 @@ void CpGrid::addLgrsUpdateLeafView(const std::vector<std::array<int,3>>& cells_p
     std::vector<int> all_patch_corners = (*data_[0]).getPatchCorners(startIJK_vec[0], endIJK_vec[0]);
     std::vector<int> all_patch_faces = (*data_[0]).getPatchFaces(startIJK_vec[0], endIJK_vec[0]);
     std::vector<int> all_patch_cells = (*data_[0]).getPatchCells(startIJK_vec[0], endIJK_vec[0]);
+    // Check all the cells to be refined have no NNC (no neighbouring connections).
+    if ((*data_[0]).hasNNCs(all_patch_cells)){
+        if (comm().rank()==0){
+            OPM_THROW(std::logic_error, "NNC face on a cell containing LGR is not supported yet.");
+        }
+        else{
+            OPM_THROW(std::logic_error, "NNC face on a cell containing LGR is not supported yet.");
```

Comment:
Somehow my comment vanished or I am simply to dumb to comment. Next try:

we do the same here in if and else branch. Maybe just skip the if-else and just throw?

Reply:
I see. In  any case, there is a typo. The second throw should have been OPM_THROW_NOLOG (as the other if-else checks).